packages/design/src/FormManager/FormEdit/PreviewPattern.tsx

```diff
@@ -49,3 +49,26 @@ export const PreviewPattern: PatternComponent = function PreviewPattern(props) {
     </div>
   );
 };
+
+/**
+ * Determine if this mouse event corresponds to a pattern edit control, which
+ * should not trigger setting the pattern edit focus.
+ */
+const isPatternEditControlEvent = (
```

Comment:
@natashapl This checks if we should treat the click event as something to change focus by looking for the `data-pattern-edit-control` attribute on the DOM node (or it's nearby parents).